### ISSUE TITLE
Build docker image failed

### ISSUE THREAD

**Issue by ziggy4git** [robert-dodier/maxima-jupyter#105]
Hello,

Getting an error when building the docker image :
`$ git clone https://github.com/robert-dodier/maxima-jupyter`
`$ cd maxima-jupyter/`
`$ docker build --tag=maxima-jupyter .`

Error in step 12:
`
Step 19/22 : RUN export PYTHON_SITE=$(python -c 'import site; print(site.getsitepackages()[0])') &&     cp maxima_lexer.py ${PYTHON_SITE}/pygments/lexers/ &&     patch ${PYTHON_SITE}/pygments/lexers/_mapping.py pygments-mapping-patch
 ---> Running in 072fd1e5aff0
patching file /usr/lib/python3.9/site-packages/pygments/lexers/_mapping.py
Hunk #1 FAILED at 217.
1 out of 1 hunk FAILED -- saving rejects to file /usr/lib/python3.9/site-packages/pygments/lexers/_mapping.py.rej
The command '/bin/sh -c export PYTHON_SITE=$(python -c 'import site; print(site.getsitepackages()[0])') &&     cp maxima_lexer.py ${PYTHON_SITE}/pygments/lexers/ &&     patch ${PYTHON_SITE}/pygments/lexers/_mapping.py pygments-mapping-patch' returned a non-zero code: 1
`

Thank you,

**Comment by robert-dodier** [comment]
Looks like the patch for pygments (which is used to colorize Maxima code in generated documents) is failing. I recently submitted the code for the Maxima colorizer to the pygments project, so the patch isn't needed anymore. 

I'll try to update the docker configuration so that it doesn't try to patch pygments. I'll also remove any CodeMIrror patching (another colorizer, this time for code in the browser) since I have published the CodeMirror mode for Maxima on npmjs.org (per instructions from CodeMirror).
